src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Program.cs

```diff
@@ -171,6 +175,12 @@ static CliRootCommand CreateCommandLineParser()
         Required = false
     };
 
+    var pipeNameOption = new CliOption<string>("--pipe")
+    {
+        Description = "The name of the pipe to use for RPC with the parent process.",
+        Required = true
```

Comment:
Should we still allow for stdin/stdout for a bit longer, if only so that way we don't have to merge/flow this simultaneously with the VS Code side of things?

Reply:
David and I discussed this, and determined it would be fine to merge both PRs close together. The only risk is that someone else updates roslyn version before I get [PR](https://github.com/dotnet/vscode-csharp/pull/6282 ) in, correct?